docs/auth0_actions_create.md

```diff
@@ -7,12 +7,26 @@ Create a new action.
 
 To create interactively, use `auth0 actions create` with no flags.
 
-To create non-interactively, supply the action name, trigger, secrets and dependencies through the flags.
+To create non-interactively, supply the action name, trigger, code, secrets and dependencies through the flags.
 
+## Usage
 ```
 auth0 actions create [flags]
 ```
 
+## Examples
```

Comment:
Moved the "Examples" up and under the "Usage" header to stand out more and to be more similar to the terminal help output.